pom.xml

```diff
@@ -2421,7 +2421,7 @@ flexible messaging model and an intuitive client API.</description>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
         <pulsar.broker.compiler.release/>
-        <pulsar.client.compiler.release/>
+        <pulsar.client.compiler.release>17</pulsar.client.compiler.release>
```

Comment:
You can remove this profile.

Reply:
Oh yes, correct. I have removed this profile.